spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -436,7 +440,12 @@ def _measure_tissue_bridges(self, im_lesion_data, p_lst, idx):
         # Note: we use [0] for the R-L direction as the orientation is RPI
         sagittal_lesion_slices = np.unique(np.where(im_lesion_data)[0])
         if self.verbose == 2:
-            printv('  Slices with lesion: ' + str(sagittal_lesion_slices), self.verbose, type='info')
+            # Convert the sagittal slices from RPI to the original orientation
+            # 'x' because R-L direction (first in RPI --> 'x')
+            sagittal_lesion_slices_print = [rpi_slice_to_orig_orientation(dim, self.orientation, slice, 'x') for slice in sagittal_lesion_slices]
+            # Reverse ordering
+            sagittal_lesion_slices_print = sagittal_lesion_slices_print[::-1]
+            printv('  Slices with lesion: ' + str(sagittal_lesion_slices_print), self.verbose, type='info')
```

Comment:
Why do we reverse the ordering here? o:

Reply:
Just to sort:
`sagittal_lesion_slices_print` --> `[13, 12, 11, 10, 9]`
`sagittal_lesion_slices_print[::-1]` --> `[9, 10, 11, 12, 13]`